selfdrive/modeld/modeld.cc

```diff
@@ -210,7 +212,9 @@ int main(int argc, char **argv) {
       }
 
     }
-    visionbuf_free(&yuv_ion);
+    for(auto &v : cl_buffers) {
+      clReleaseMemObject(v.second);
```

Comment:
is clReleaseMemObject all we need here? visionbuf_free also calls munmap and closes the buffer's file descriptor

Reply:
On the receiving end we don't have a fd to the ion buffer handle to close